tekton/ci/templates/plumbing-template.yaml

```diff
@@ -416,7 +416,7 @@ spec:
         - name: gitHubCommand
           value: $(tt.params.gitHubCommand)
         - name: folders
-          value: bots/buildbot/config bots/mariobot/config pipelinerun-logs/config boskos gubernator prow robocat tekton
+          value: bots/buildconfig/config bots/mariobot/config pipelinerun-logs/config boskos gubernator prow robocat tekton
```

Comment:
should this be `buildconfig` or `buildcaptain`?

Reply:
whoa thanks for catching that!